tests/conftest.py

```diff
@@ -0,0 +1,73 @@
+# test/conftest.py
+import json
+import os
+import urllib
+
+import pytest
+import vcr
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
+@pytest.fixture
+def my_vcr():
+    my_vcr = vcr.VCR(
+        before_record_request=vcr_scrub_reqeust,
+        before_record_response=vcr_scrub_response,
+        decode_compressed_response=True,
+        filter_post_data_parameters=[
+            ("email", "user@example.com"),
+            ("password", "password"),
+        ],
+        filter_headers=[("Cookie", "JSESSIONID=sessioncookie")],
+    )
+    return my_vcr
+
+
+@pytest.fixture
+def vcr_env():
+    if os.getenv("DSPACE_PYTHON_CLIENT_ENV") == "vcr_create_cassettes":
+        env = {
+            "url": os.environ["TEST_DSPACE_API_URL"],
+            "email": os.getenv("TEST_DSPACE_EMAIL"),
+            "password": os.getenv("TEST_DSPACE_PASSWORD"),
+        }
+    else:
+        env = {
+            "url": "https://dspace-example.com/rest",
+            "email": "user@example.com",
+            "password": "password",
+        }
+    return env
+
+
+def vcr_scrub_reqeust(request):
+    """Replaces the request URI with fake data"""
+    split_uri = urllib.parse.urlsplit(request.uri)
+    new_uri = urllib.parse.urljoin("https://dspace-example.com", split_uri.path)
+    request.uri = new_uri
+    return request
+
+
+def vcr_scrub_response(response):
```

Comment:
These things are always a pain to get setup but are worth it longterm to not have to manually scrub cassettes. 👍🏻